vmngclient/api/templates/feature_template.py

```diff
@@ -45,3 +46,30 @@ def payload_path(self) -> Path:
     @property
     def type(self) -> str:
         raise NotImplementedError()
+
+    @classmethod
+    def get(cls, session: vManageSession, name: str) -> FeatureTemplate:
+        """Gets feature template model corresponding to existing feature template based on provided name
+
+        Args:
+            session: vManageSession
+            name: name of the existing feature template
+
+        Returns:
+            FeatureTemplate: filed out feature template model
+        """
+        from vmngclient.utils.feature_template import choose_model, find_template_values
```

Comment:
move import on the beginning of the file

Reply:
cannot do this, circular import created

Reply:
I would try to find a design in which there are no circular buffers, maybe modules could be arranged better?

Reply:
I would try to find a design in which there are no circular imports, maybe modules could be arranged better?